test/ContainerTest.php

```diff
@@ -6,15 +6,22 @@
 
 use Laminas\ContainerConfigTest\AbstractMezzioContainerConfigTest;
 use Laminas\ContainerConfigTest\SharedTestTrait;
+use Laminas\ServiceManager\ConfigInterface;
 use Laminas\ServiceManager\ServiceManager;
 use Psr\Container\ContainerInterface;
 
+/**
+ * @see ConfigInterface
+ *
+ * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
+ */
 class ContainerTest extends AbstractMezzioContainerConfigTest
 {
     use SharedTestTrait;
 
     protected function createContainer(array $config): ContainerInterface
     {
+        /** @psalm-suppress MixedArgumentTypeCoercion */
```

Comment:
The base `createContainer` method should use the `ServiceManagerConfigurationType`, but I see it's imported here, and never used

Reply:
I can either suppress this error or `MoreSpecificImplementedParamType` due to the implementation details here: 
https://github.com/laminas/laminas-container-config-test/blob/6a7c3f3fed29ed844cdc22b1d911cb2ef745873e/src/AbstractContainerTest.php#L18

I don't know what would be better tho.

Reply:
This was more a note on the fact that the imported type in this file is not used. The class-level docblock can probably be dropped

Reply:
Ah yes, I've added it initially but then removed it again. Will drop that.